dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -708,6 +708,10 @@ warning.earlyRelease=CWWKE0953W: This version of {0} is an unsupported early rel
 warning.earlyRelease.explanation=New features that are not fully tested are made available in early releases to elicit feedback before they are included in a supported release.
 warning.earlyRelease.useraction=Do not use an early release version on production systems, since early release versions are not supported.
 
+error.invalidPhaseName=CWWKE0954E: An empty or unknown checkpoint phase "{0}" was specified.
```

Comment:
I prefer this:

The specified checkpoint phase "{0}" is empty or unknown.

Reply:
Writers are directed to use noun identifiers after files, variables, commands etc. For example, "The {0} URL", "the {1} file". We break this rule when the variable being inserted would cause the sentence to be awkward. If you feel that that is the case here, I'll defer.

Reply:
OK, then I will go with:

The "{0}" checkpoint phase specified is empty or unknown.